cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -6,6 +6,7 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE QuantifiedConstraints #-}
```

Comment:
Is this necessary?

Reply:
It might be not, since there's `ShowQuery` wrapper.

Reply:
Fixed.